src/DynamoCore/Search/SearchElements/ZeroTouchSearchElement.cs

```diff
@@ -23,6 +23,11 @@ public class ZeroTouchSearchElement : NodeSearchElement
         public override string CreationName { get { return functionDescriptor != null ? functionDescriptor.MangledName : this.Name; } }
 
         /// <summary>
+        /// The name that is used for Web UI
+        /// </summary>
+        public override string FullyQualifiedName { get { return functionDescriptor != null ? functionDescriptor.QualifiedName : this.Name; } }
```

Comment:
I checked the nodes, so far I have yet to find a node that has `null` as its `functionDescriptor`.